### PR TITLE
[Android]Added KotlinPoet group

### PR DESCRIPTION
**Pull request by NUmeroAndDev**

### CODE REVIEW

File: groupAndroidPackages.json [hunk 1]
```diff
@@ -347,6 +347,13 @@
         "org.threeten:threetenbp",
         "com.jakewharton.threetenabp:threetenabp"
       ]
+    },
+    {
+      "groupName": "KotlinPoet",
```
Comment (ikesyo) [inline groupAndroidPackages.json:352]: Please sort it alphabetically (between `Kotlin` and `kotlinx.coroutines`).

File: groupAndroidPackages.json [hunk 2]
```diff
@@ -347,6 +347,13 @@
...
+      "matchPackageNames": [
+        "com.squareup:kotlinpoet:",
+        "com.squareup:kotlinpoet-ksp:"
+      ]
```
Comment (ikesyo) [inline groupAndroidPackages.json:356]:
```suggestion
      "matchPackageNames": [
        "com.squareup:kotlinpoet",
        "com.squareup:kotlinpoet-ksp"
      ]
```

**Review by ikesyo — CHANGES_REQUESTED** [review]

File: groupAndroidPackages.json (hunk 1 above)
Comment (NUmeroAndDev) [reply groupAndroidPackages.json]: Fixed : https://github.com/hatena/renovate-config/pull/54/commits/c3cb3031b9530fdc270735d2ad477315387e6ffe

File: groupAndroidPackages.json (hunk 2 above)
Comment (NUmeroAndDev) [reply groupAndroidPackages.json]: Thanks!

**Review by ikesyo — APPROVED** [review]
LGTM 👍

**Pull request merged**